sdk/eventhub/Azure.Messaging.EventHubs/src/Metadata/LastEnqueuedEventProperties.cs

```diff
@@ -9,7 +9,7 @@ namespace Azure.Messaging.EventHubs.Metadata
     ///   A set of information about the enqueued state of a partition, as observed by the consumer.
     /// </summary>
     ///
-    public class LastEnqueuedEventProperties
+    public struct LastEnqueuedEventProperties
```

Comment:
I see you changed `LastEnqueuedEventProperties` from `class` to `struct`. Shouldn't we follow the [struct guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/struct) here?

Reply:
The only missing bit, as far as I know, was equability - which isn't something that we wanted to support for these.    That said, I just changed it back to `class` because I wanted to limit the construction to `protected` and `internal` like the other metadata types.